xml/System.Runtime.InteropServices/CreateComInterfaceFlags.xml

```diff
@@ -39,7 +39,16 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The caller will provide an IUnknown Vtable.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is useful in scenarios when the caller has no need to rely on an IUnknown instance that is used when running managed code is not possible (i.e. during a GC). In traditional COM scenarios this is common, but scenarios involving <see href="https://docs.microsoft.com/windows/win32/api/windows.ui.xaml.hosting.referencetracker/nn-windows-ui-xaml-hosting-referencetracker-ireferencetrackertarget">Reference Tracker hosting</see> calling of the IUnknown API during a GC is possible.
```

Comment:
```suggestion
This flag is useful in scenarios when the caller has no need to rely on an IUnknown instance that's used when it's not possible to run managed code (that is, during a garbage collection). This is common in traditional COM scenarios, but scenarios where <see href="https://docs.microsoft.com/windows/win32/api/windows.ui.xaml.hosting.referencetracker/nn-windows-ui-xaml-hosting-referencetracker-ireferencetrackertarget">Reference Tracker hosting</see> calls the IUnknown API during a garbage collection are possible.
```

Reply:
Based on @adegeo 's [suggestion](https://github.com/dotnet/dotnet-api-docs/pull/4286#discussion_r465233882), it seems I can remove the domain form the URL, so I'm copying your suggestion and making that small change:

```suggestion
This flag is useful in scenarios when the caller has no need to rely on an IUnknown instance that's used when it's not possible to run managed code (that is, during a garbage collection). This is common in traditional COM scenarios, but scenarios where <see href="/windows/win32/api/windows.ui.xaml.hosting.referencetracker/nn-windows-ui-xaml-hosting-referencetracker-ireferencetrackertarget">Reference Tracker hosting</see> calls the IUnknown API during a garbage collection are possible.
```